core/clingutils/src/TClingUtils.cxx

```diff
@@ -1744,7 +1744,7 @@ bool ROOT::TMetaUtils::ExtractAttrIntPropertyFromName(const clang::Decl& decl,
 }
 
 ////////////////////////////////////////////////////////////////////////////////
-/// FIXME: a function of ~300 lines!
+/// FIXME: a function of 450+ lines!
```

Comment:
LOL :-)

Reply:
nothing worse than a comment out of sync with the code :P